source/features/link-to-github-io.tsx

```diff
@@ -30,15 +30,16 @@ function initRepo(signal: AbortSignal): void {
 }
 
 function initRepoList(signal: AbortSignal): void {
-	observe('a[href$=".github.io"][itemprop="name codeRepository"]', addRepoListLink, {signal});
+	// Also consider any old GitHub Pages repo like: resume/resume.github.com , issue: https://github.com/refined-github/refined-github/issues/6228
+	observe('a:is([href$=".github.io"], [href$=".github.com"])[itemprop="name codeRepository"]', addRepoListLink, {signal});
 }
 
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.hasRepoHeader,
 	],
 	asLongAs: [
-		() => Boolean(getRepo()?.name.endsWith('.github.io')),
+		() => Boolean(getRepo()?.name.endsWith('.github.io') || getRepo()?.name.endsWith('.github.com')),
```

Comment:
Let's use a regex instead
```suggestion
		() => /\.github\.(io|com).test(getRepo()?.name),
```